sdk/servicebus/azure-messaging-servicebus/migration-guide.md

```diff
@@ -75,19 +75,19 @@ and namespaces that begin with `com.microsoft.azure.servicebus` and a version of
 
 ### Client hierarchy
 
-As part of the new Java SDK guidelines, all clients are instantiated from a builder. Each builder can instantiate an
-async client or a sync client via `buildAsyncClient()` or `buildClient()`. In our new library, the single entry point is
-via `ServiceBusClientBuilder`, rather than one for each of queue, topic, subscription, and session. You can create
-senders and receivers from this client to the queue/topic/subscription/session of your choice and start
-sending/receiving messages.
+As part of the new Java SDK guidelines, all clients are instantiated from a builder which is the single entry point to the library.
+Each client is expected to have a sync and async version that can be instantiated via `buildAsyncClient()` or `buildClient()` methods
+on the builder.
```

Comment:
Just splitting the "common" things we are doing for all Track 2 SDKs and what we did for Service Bus specifically